lib/jest-helpers.js

```diff
@@ -0,0 +1,40 @@
+'use strict'
+
+const nunjucks = require('nunjucks')
+const cheerio = require('cheerio')
+
+const COMPONENT_DIR = 'src/components'
+
+nunjucks.configure(COMPONENT_DIR, {
+  trimBlocks: true,
+  lstripBlocks: true
+})
+
+function render (name, params) {
+  const output = nunjucks.render(name + '/template.njk', { params })
```

Comment:
What happens if you try to render a non-existent component? Is the error message clear? If not, can we add error handling to this function.

Reply:
Will check this, good shout.

I'll write a test for this...

Reply:
``` 
FAIL  src/components/back-link/template.test.js
  back-link component
    ✕ renders the default with an anchor, href and text correctly (5ms)
    ✓ renders classes correctly (12ms)
    ✓ renders custom text correctly (3ms)
    ✓ renders html correctly (1ms)
    ✓ renders attributes correctly (3ms)
    ○ skipped 1 test

  ● back-link component › renders the default with an anchor, href and text correctly

    template not found: back-links/template.njk
```